### ISSUE TITLE
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

### ISSUE THREAD

**Issue by deathvu** [telegramdesktop/tdesktop#17025]
### Steps to reproduce

Download telegram-desktop for linux x64
Extract tsetup.xz
Execute "Telegram"

### Expected behaviour

telegram-desktop runs

### Actual behaviour

error:
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

### Operating system

Linux, Debian 10 (buster) x64, dwm X11 glibc 2.33

### Version of Telegram Desktop

3.1.1

### Installation source

Static binary from official website

**Comment by deathvu** [comment]
> This error is not critical, no need to report it

so how to solve it

**Comment by ilya-fedin** [comment]
Solve what? :thinking: 

Looks like you really poorly described the problem in your first post...

**Comment by deathvu** [comment]
> This error is not critical, no need to report it

so how to solve it

> Solve what? 🤔
> 
> Looks like you really poorly described the problem in your first post...


there is only one err message after start: [ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

telegram version 3.1.1 (i have tried with 2.7.3, 3.1.0, 3.1 beta - same result)
my env: Linux, Debian 10 (buster) x64, dwm X11 glibc 2.33


$ ./Telegram
$ ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

**Comment by ilya-fedin** [comment]
As I said, this error is not critical, no need to solve it

**Comment by deathvu** [comment]
> As I said, this error is not critical, no need to solve it. Just ignore it.

oh... sorry, i haven't mentioned that telegram doesnt start

after err message appears nothing happens

**Comment by ilya-fedin** [comment]
> oh... sorry, i haven't mentioned that telegram doesnt start

Ok, you need to provide the log.txt (~/.local/share/TelegramDesktop/log.txt)

**Comment by ilya-fedin** [comment]
> cannot dowload any file, cannot open any link

That sound like you don't have `xdg-desktop-portal` (the frontend) or one of its backends (`xdg-desktop-portal-kde` or `xdg-desktop-portal-gtk`, depending on what dialogs you like more)

**Comment by deathvu** [comment]
> > cannot dowload any file, cannot open any link
> 
> That sounds like you don't have `xdg-desktop-portal` (the frontend) or one of its backends (`xdg-desktop-portal-kde` or `xdg-desktop-portal-gtk`, depending on which dialogs you like more)

oh, the problem is my de is not "standard" one. It is DWM from suckless
p.s log file will be later, thank you

**Comment by ilya-fedin** [comment]
> oh, the problem is my de is not "standard" one. It is DWM from suckless

You can still install backends of other DEs

**Comment by ilya-fedin** [comment]
> after err message appears nothing happens

Does it crash or just hangs?

**Comment by deathvu** [comment]
> > cannot dowload any file, cannot open any link
> 
> That sounds like you don't have `xdg-desktop-portal` (the frontend) or one of its back

oh, the problem is my de is not "standard" one. It is DWM from suckless
p.s log file will be later, thank you

> > oh, the problem is my de is not "standard" one. It is DWM from suckless
> 
> You can still install backends of other DEs

okay, i will try, thank you

app just hands

**Comment by ilya-fedin** [comment]
Why are you repeating your previous last message?

**Comment by pefu** [comment]
I've done some research on the message and found this issue:
https://github.com/kcat/openal-soft/issues/554
**TL;DR-Summary**: As long as you don't use audio you can safely ignore this message. Otherwise you could try to use the command:
`usermod -G realtime -a $(whoami)`
and see if it fixes the issue.
